simulator/source/Combat_simulator.cpp

```diff
@@ -452,6 +452,33 @@ void Combat_simulator::mortal_strike(Sim_state& state)
     logger_.print("Current rage: ", int(rage));
 }
 
+void Combat_simulator::devastate(Sim_state& state)
+{
+    //TODO: Add DPR support
+    logger_.print("Devastate!");
+    int bonus = (config.exposed_armor) ? 0 : sunder_armor_stacks_ * 35;
+    double damage = (state.main_hand_weapon.normalized_swing(state.special_stats)/2 + bonus);
```

Comment:
Instead of adding a new type of swing for "no bonus damage swing" or something like that I opted to just subtract the bonus damage back out of the result from the normalized swing function.

Reply:
I wonder if Devastate is affected by Expose Weakness and Improved Hunter's Mark. According to Emrus, it's most likely scripted, and might have any number of oversights ;)

Reply:
I have a feeling there's multiple strange interactions like this that probably aren't handled correctly. I'd be happy to help test this on PTR if you want though!

Reply:
If you have a hunter at your disposal: sure. My subscription has expired again ;)